### PR TITLE
Smaller items for 24.06

### PR DESCRIPTION
**Pull request by mikemckiernan**
Review HTML
RN: https://nvidia.github.io/cloud-native-docs/review/pr-73/gpu-operator/latest/release-notes.html#v24-6-0
Component Matrix: https://nvidia.github.io/cloud-native-docs/review/pr-73/gpu-operator/latest/platform-support.html#gpu-operator-component-matrix

### CODE REVIEW

File: gpu-operator/life-cycle-policy.rst [hunk 1]
```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
        | `535.183.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-183-01/index.html>`_,
        | `470.256.02 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-470-256-02/index.html>`_
 
    * - NVIDIA Driver Manager for K8s
-     - `v0.6.8 <https://ngc.nvidia.com/catalog/containers/nvidia:cloud-native:k8s-driver-manager>`__
+     - `v0.6.10 <https://ngc.nvidia.com/catalog/containers/nvidia:cloud-native:k8s-driver-manager>`__
 
    * - NVIDIA Container Toolkit
-     - `1.15.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
+     - `1.16.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
```
Comment (cdesiniotis) [inline gpu-operator/life-cycle-policy.rst:100]: We released a patch release recently to resolve 4757132
```suggestion
     - `1.16.1 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
```

File: gpu-operator/life-cycle-policy.rst [hunk 2]
```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
 
    * - NVIDIA Kubernetes Device Plugin
-     - `0.15.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
+     - `0.16.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
```
Comment (cdesiniotis) [inline gpu-operator/life-cycle-policy.rst:103]: We released a patch release recently to resolve 4757132
```suggestion
     - `0.16.1 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
```

File: gpu-operator/life-cycle-policy.rst [hunk 3]
```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
+     - `3.3.6-3.4.2 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
 
    * - Node Feature Discovery
-     - v0.15.4
+     - v0.16.3
 
    * - | NVIDIA GPU Feature Discovery
        | for Kubernetes
-     - `0.15.0 <https://github.com/NVIDIA/gpu-feature-discovery/releases>`__
+     - `0.16.0 <https://github.com/NVIDIA/gpu-feature-discovery/releases>`__
```
Comment (cdesiniotis) [inline gpu-operator/life-cycle-policy.rst:113]: Note, we converged the device-plugin and gfd repositories. So all future development and releases will occur at the device-plugin repository.
```suggestion
     - `0.16.1 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
```

File: gpu-operator/life-cycle-policy.rst [hunk 4]
```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
 
    * - NVIDIA MIG Manager for Kubernetes
-     - `0.7.0 <https://github.com/NVIDIA/mig-parted/tree/main/deployments/gpu-operator>`__
+     - `0.8.0 <https://github.com/NVIDIA/mig-parted/tree/main/deployments/gpu-operator>`__
 
    * - DCGM
-     - `3.3.5-1 <https://docs.nvidia.com/datacenter/dcgm/latest/release-notes/changelog.html>`__
+     - `3.3.6-1 <https://docs.nvidia.com/datacenter/dcgm/latest/release-notes/changelog.html>`__
```
Comment (cdesiniotis) [inline gpu-operator/life-cycle-policy.rst:119]:
```suggestion
     - `3.3.7-1 <https://docs.nvidia.com/datacenter/dcgm/latest/release-notes/changelog.html>`__
```

File: gpu-operator/life-cycle-policy.rst [hunk 5]
```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
-     - `1.15.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
+     - `1.16.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
 
    * - NVIDIA Kubernetes Device Plugin
-     - `0.15.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
+     - `0.16.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
 
    * - DCGM Exporter
-     - `3.3.5-3.4.1 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
+     - `3.3.6-3.4.2 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
```
Comment (cdesiniotis) [inline gpu-operator/life-cycle-policy.rst:106]: We just picked up the new release of DCGM and DCGM exporter this week
```suggestion
     - `3.3.7-3.5.0 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
```

**Comment by mikemckiernan** [comment]
@tariq1890 , when you can, PTAL at what I typed for the RBAC work: https://nvidia.github.io/cloud-native-docs/review/pr-73/gpu-operator/latest/release-notes.html#fixed-issues

**Comment by tariq1890** [comment]
@mikemckiernan They look good to me, Thanks!

File: gpu-operator/life-cycle-policy.rst [hunk 6]
```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
-     - `1.15.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
+     - `1.16.1 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
 
    * - NVIDIA Kubernetes Device Plugin
-     - `0.15.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
+     - `0.16.1 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
 
    * - DCGM Exporter
-     - `3.3.5-3.4.1 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
+     - `3.3.7-3.5.0 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
```
Comment (tariq1890) [inline gpu-operator/life-cycle-policy.rst:106]:
```suggestion
     - `3.3.7-3.5.0 <https://github.com/NVIDIA/dcgm-exporter/releases>`__
```

File: gpu-operator/release-notes.rst [hunk 7]
```diff
@@ -34,6 +34,94 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
...
+    - NVIDIA GDRCopy Driver v2.4.1-1
+
+* Added support for NVIDIA Network Operator v24.4.0.
+  Refer to :ref:`Support for GPUDirect RDMA` and :ref:`Support for GPUDirect Storage`.
+
+* Added support for using precompiled driver containers that use the 550 branch Open Kernel module drivers.
+  Specify ``--set driver.useOpenKernelModules=true --set driver.usePrecompiled=true --set driver.version=550``
+  when you install or upgrade the Operator.
+  Support remains limited to Ubuntu 22.04.
+  Refer to :doc:`precompiled-drivers` for more information.
```
Comment (cdesiniotis) [inline gpu-operator/release-notes.rst:74]: A few comments:
1) This change is not tied to a particular driver version. It is included in both the R535 and R550 variants of the container images.
2) This change is only included in precompiled container images published starting on 7/15, see https://catalog.ngc.nvidia.com/orgs/nvidia/containers/driver/tags. As we do not update previously pushed tags, all previous tags will not have this change. I am not sure if we should explicitly mention it, but here are the first tags which include this change (and all future versions will): 
```
<driver-branch>-5.15.0-116-generic-ubuntu22.04
<driver-branch>-5.15.0-1060-nvidia-ubuntu22.04
<driver-branch>-5.15.0-1063-oracle-ubuntu22.04
<driver-branch>-5.15.0-1068-azure-ubuntu22.04
<driver-branch>-5.15.0-1065-aws-ubuntu22.04
```

File: gpu-operator/release-notes.rst [hunk 8]
```diff
@@ -34,6 +34,94 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
...
+
+  * For H20 devices:
+
+    * ``1g.12gb``
+    * ``2g.24gb``
+    * ``3g.48gb``
+
+.. _v24.6.0-fixed-issues:
+
+Fixed Issues
```
Comment (cdesiniotis) [inline gpu-operator/release-notes.rst:96]: Let's add bullets for the following:

https://github.com/NVIDIA/gpu-operator/issues/782
- This was a bug with NFD. If the NFD worker pod was deleted / restarted on a node, all NFD node labels were removed from the node, therefore leading to all GPU Operator operands to get restarted. 

https://github.com/NVIDIA/gpu-operator/issues/761
- This bug was described here: https://github.com/NVIDIA/cloud-native-docs/pull/76

Comment (cdesiniotis) [reply gpu-operator/release-notes.rst:96]: Here is a possible improvement we can call out: https://github.com/NVIDIA/gpu-operator/issues/802

File: gpu-operator/release-notes.rst (hunk 7 above)
Comment (mikemckiernan) [reply gpu-operator/release-notes.rst:74]: I can't recall what gave me the impression it began with 550, but this is all good information and valuable in release notes.  TY!

File: gpu-operator/release-notes.rst [hunk 9]
```diff
@@ -34,6 +34,94 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
...
+    * ``2g.35gb``
+    * ``3g.71gb``
+    * ``4g.71gb``
+    * ``7g.141gb``
+
+  * For H20 devices:
+
+    * ``1g.12gb``
+    * ``2g.24gb``
+    * ``3g.48gb``
```
[the remark this answers is not in the document]
Comment (mikemckiernan) [reply gpu-operator/release-notes.rst:92]: Aha!  I have a revision.  Let's look it over and adjust when we speak later.  TY!

File: gpu-operator/life-cycle-policy.rst [hunk 10]
```diff
@@ -86,60 +86,58 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
 
    * - NVIDIA GPU Operator
-     - v24.3.0
+     - v24.6.0
 
    * - NVIDIA GPU Driver
-     - | `550.90.07 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-90-07/index.html>`_ (recommended),
-       | `550.54.15 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-54-15/index.html>`_ (default),
-       | `535.183.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-183-01/index.html>`_,
-       | `470.256.02 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-470-256-02/index.html>`_
```
Comment (mikemckiernan) [inline gpu-operator/life-cycle-policy.rst:95]: @cdesiniotis , @tariq1890  reminded me in the DMA-BUF update that 470 is EOL at the end of this month.  I removed it from this table, but have no objection to restoring it if that's preferred.

File: gpu-operator/life-cycle-policy.rst [hunk 11]
```diff
@@ -86,60 +86,58 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
...
+     - | `550.90.07 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-90-07/index.html>`_ (default),
+       | `535.183.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-183-01/index.html>`_
```
Comment (tariq1890) [inline gpu-operator/life-cycle-policy.rst:93]: Oh missed this. The latest driver for R535 is not `535.183.06`
```suggestion
       | `535.183.06 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-183-06/index.html>`_
```

File: gpu-operator/precompiled-drivers.rst [hunk 12]
```diff
@@ -59,9 +59,9 @@ Limitations and Restrictions
   refer to :external+ocp:doc:`gpu-operator-with-precompiled-drivers`.
 
 * NVIDIA supports precompiled driver containers for the most recently released long-term
-  servicing branch (LTSB) driver branch, 525.
+  servicing branch (LTSB) driver branch.
 
-* NVIDIA builds images for the ``generic`` kernel variant.
+* NVIDIA builds images for the ``aws``, ``azure``, ``generic``, ``nvidia``, and ``oracle`` kernel variants.
```
Comment (tariq1890) [inline gpu-operator/precompiled-drivers.rst:64]: I would prefer using the term flavour/flavor here.  As per this [link](https://ubuntu.com/kernel/variants), the suffix is the flavour, but the whole kernel version string is called the variant
```suggestion
* NVIDIA builds images for the ``aws``, ``azure``, ``generic``, ``nvidia``, and ``oracle`` kernel variants.
```

File: gpu-operator/release-notes.rst [hunk 13]
```diff
@@ -34,6 +34,113 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
...
+    * ``3g.48gb`` :math:`\times` 1
+
+.. _v24.6.0-fixed-issues:
+
+Fixed Issues
+------------
+
+* Role-based access controls for the following components were reviewed and revised to use least-required privileges:
+
+  * GPU Operator
```
Comment (tariq1890) [inline gpu-operator/release-notes.rst:112]: RBAC cleanup was also performed for the following components:

- GPU Feature Discovery
- K8s Device Plugin
- KubeVirt Device Plugin
- vGPU Host Manager

File: gpu-operator/life-cycle-policy.rst (hunk 10 above)
Comment (tariq1890) [reply gpu-operator/life-cycle-policy.rst:95]: I think this is fine as-is, thanks!

File: gpu-operator/precompiled-drivers.rst (hunk 12 above)
Comment (mikemckiernan) [reply gpu-operator/precompiled-drivers.rst:64]: "Flavour" appears once on that page and really seems like a colloquialism.  I can't count how many times "variant" is used and the table of Current variant kernels shows the aws and azure strings.  I'd really prefer to continue with variant.

**Pull request merged**